src/sentry/static/sentry/app/components/lightWeightNoProjectMessage.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```

Comment:
Not sure if I should have just wrote some logic into the normal `noProjectMessage.jsx` to handle a `lightweight` prop. Which is preferred? It seems like writing tests would be easier I just added a `lightweight` prop to `noProjectMessage.jsx`

Reply:
Yeah, I think it's better to just add a `lightweight` prop, maybe `detailed` to match our API (and just add a default prop).